index.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+  equals: 'equals',
+  concat: 'concat',
+  empty: 'empty',
+  map: 'map',
+  ap: 'ap',
+  of: 'of',
+  reduce: 'reduce',
+  sequence: 'sequence',
+  chain: 'chain',
+  extend: 'extend',
+  extract: 'extract'
+}
```

Comment:
This is supposed to be the first backward-compatibility version, that simply uses current names and doesn't prevent name clashes. All current libs will be automatically compatible with this version, I've wrote about this in https://github.com/fantasyland/fantasy-land/issues/92#issuecomment-172357516

Then we can release a second version that uses unique names. I suggest to use strings (something like `fantasy-land/map`) though, really don't see any advantages of using Symbols, and there are number of disadvantages as @robotlolita pointed out in https://github.com/fantasyland/fantasy-land/issues/92#issuecomment-139907109